lib/scss/components/news-node/_news-node.scss

```
@@ -164,6 +164,18 @@
     span {
       @include sr-only;
     }
+
+    &.su-link--external {
+      &::after {
+        @include hide-visually;
+      }
+    }
+
+    & svg {
```

```suggestion
    svg {
```

Don't need the & here.

---

Eliminated.